azurerm/resource_arm_iothub.go

```diff
@@ -323,6 +323,32 @@ func resourceArmIotHub() *schema.Resource {
 				},
 			},
 
+			"ip_filter_rule": {
+				Type:     schema.TypeList,
```

Comment:
as there could be multiple of these (and they could be returned in any order) - I think this could be better as a Set?

```suggestion
				Type:     schema.TypeSet,
```